src/components/pages/farmer-member/show/farmer-info.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  Box,
+  Flex,
+  Heading,
+  Image,
+  Stack,
+  Table,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@chakra-ui/react";
+import dynamic from "next/dynamic";
+import React from "react";
+
+import FARM_LAND_IMAGE from "../../../../../public/assets/farm-land-default.jpg";
```

Comment:
There is a neater way of importing in the codebase instead of long relative imports. Get in touch with me regarding this so that we can see if that's feasible here.